build.xml

```diff
@@ -50,7 +50,8 @@
             <arg path="src"/>
             <arg value="xml"/>
             <arg value="phpmd.xml"/>
-            <arg value="--exclude */Tests/*"/>
+            <arg value="--exclude"/>
```

Comment:
Ik geloof dat je hier ook `<arg line="--exclude */Tests/*"/>` voor kunt gebruiken. Dat is misschien leesbaarder dan het uitsplitsen alsof het twee losse argumenten zijn?